mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_759 net/minecraft/client/render/FirstPersonRenderer
+CLASS net/minecraft/class_759 net/minecraft/client/render/Item/HeldItemRenderer
```

Comment:
This casing is still a problem... needs a git client to fix this really

Reply:
Yeah, I don't know what is going on here. I fixed this before as well.